docs/UserGuide.md

```diff
@@ -68,7 +73,7 @@ Amount entered must be greater than 0 when rounded to 2dp. <br>
 
 Example valid inputs: <br>
 - `50` <br>
-- `1500` <br>
+- `1500.99` <br>
```

Comment:
can add a 1.005 here (it'll round to 1.01

Reply:
but logically currency we do not put to 3dp like normally, already stated that it is rounded values

Reply:
i think we can still add as valid input, then state that any input greater than 2dp wil be rounded off to 2dp

Reply:
done!